modules/JsonExporter/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>gephi-plugin-parent</artifactId>
+        <groupId>org.gephi</groupId>
+        <version>0.9.0</version>
+    </parent>
+
+    <groupId>org.gephi</groupId>
```

Comment:
Thank you @mbastian for this note and the others. I will make all the changes and submit an updated pull request. I thought org.gephi would be reserved, but had left it in because of the {project.groupId} reference in the dependencies. I will remove that and change this groupId. Thank you for your time to review the code so thoroughly.